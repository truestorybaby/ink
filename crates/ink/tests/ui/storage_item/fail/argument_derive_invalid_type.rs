#[ink::storage_item(derive = "false")]
#[derive(Default)]
struct Contract<KEY: StorageKey = ManualKey<123>> {
    a: u16,
    b: u64,
    c: u128,
}

fn main() {}
