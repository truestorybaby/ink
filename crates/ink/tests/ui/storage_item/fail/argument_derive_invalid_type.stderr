error: expected a bool literal for `derive` ink! storage item configuration argument
 --> tests/ui/storage_item/fail/argument_derive_invalid_type.rs:1:21
  |
1 | #[ink::storage_item(derive = "false")]
  |                     ^^^^^^^^^^^^^^^^
